File: app/src/main/java/com/example/myapplication/activity/VollyDActivity.java
package com.example.myapplication.activity;

import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import com.android.volley.Request;
import com.android.volley.RequestQueue;
import com.android.volley.Response;
import com.android.volley.VolleyError;
import com.android.volley.toolbox.StringRequest;
import com.android.volley.toolbox.Volley;
import com.example.myapplication.R;

public class VollyDActivity extends AppCompatActivity {

    private RequestQueue requestQueue;
    private StringRequest stringRequest;
    Button vollyButton;
    private String url = "http://www.mocky.io/v2/597c41390f0000d002f4dbd1";
   // private String url = "https://www.unimedliving.com/api/v2/audios";
    //private String url = "https://www.unimedliving.com/api/v2/articles";
    //private String url = "https://api.androidhive.info/contacts/";
    //private String url = "https://www.unimedliving.com/api/v2/videos";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_volly_d);

        vollyButton = findViewById(R.id.vollyButton);

        vollyButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                sendAndRequestResponse();
            }
        });
    }

    private void sendAndRequestResponse() {

        requestQueue = Volley.newRequestQueue(this);
        stringRequest = new StringRequest(Request.Method.GET, url, new Response.Listener<String>() {
            @Override
            public void onResponse(String response) {
                Toast.makeText(VollyDActivity.this, "Response :  " + response.toString(), Toast.LENGTH_SHORT).show();
                Log.e("AAA","Reps"+response.toString());

            }
        }, new Response.ErrorListener() {
            @Override
            public void onErrorResponse(VolleyError error) {
                Toast.makeText(VollyDActivity.this, "Error :  " + error.toString(), Toast.LENGTH_SHORT).show();
            }
        });

        requestQueue.add(stringRequest);

    }
}

File: app/src/main/java/com/example/myapplication/activity/VollyDActivity.kt
package com.example.myapplication.activity

import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.android.volley.Request
import com.android.volley.RequestQueue
import com.android.volley.Response
import com.android.volley.toolbox.StringRequest
import com.android.volley.toolbox.Volley
import com.example.myapplication.R

class VollyDActivity : AppCompatActivity() {
    private var requestQueue: RequestQueue? = null
    private var stringRequest: StringRequest? = null
    var vollyButton: Button? = null
    private val url = "http://www.mocky.io/v2/597c41390f0000d002f4dbd1"
    // private String url = "https://www.unimedliving.com/api/v2/audios";
//private String url = "https://www.unimedliving.com/api/v2/articles";
//private String url = "https://api.androidhive.info/contacts/";
//private String url = "https://www.unimedliving.com/api/v2/videos";
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_volly_d)
        vollyButton = findViewById(R.id.vollyButton)
        vollyButton!!.setOnClickListener(View.OnClickListener { sendAndRequestResponse() })
    }

    private fun sendAndRequestResponse() {
        requestQueue = Volley.newRequestQueue(this)
        stringRequest = StringRequest(
            Request.Method.GET,
            url,
            Response.Listener { response ->
                Toast.makeText(this@VollyDActivity, "Response :  $response", Toast.LENGTH_SHORT).show()
                Log.e("AAA", "Reps$response")
            },
            Response.ErrorListener { error ->
                Toast.makeText(this@VollyDActivity, "Error :  $error", Toast.LENGTH_SHORT).show()
            })
        requestQueue!!.add(stringRequest)
    }
}